### PR TITLE
[android] add logo to android idea project

### PR DESCRIPTION
**Pull request by Arthur-GYT**
All is in the title ^^.

### CODE REVIEW

**Review by biodranik — APPROVED** [review]
Isn't this path under gitignore?

**Comment by Arthur-GYT** [comment]
> Isn't this path under gitignore?

I don't think so. When I change the icon, git detects the change

**Comment by biodranik** [comment]
This path is in gitignore: https://github.com/organicmaps/organicmaps/blob/master/.gitignore#L125

**Comment by Arthur-GYT** [comment]
@biodranik It's fixed ^^.

File: .gitignore [hunk 1]
```diff
@@ -123,6 +123,7 @@ tizen/*/.*
 tizen/*/crash-info/*
 .idea/*
 .idea
+!.idea/icon.svg
```
Comment (biodranik) [inline .gitignore:126]: `android/.idea/icon.svg` ?

**Review by biodranik — APPROVED** [review]
Thanks!

**Pull request merged**